### PR TITLE
Add the patchset of dynamic buffer pool

### PR DESCRIPTION
**Pull request by xhaihao**
0052 and 0102 are not required anymore

### CODE REVIEW

**Comment by xhaihao** [comment]
@feiwan1  @wenbinc-Bin  I deleted 0052 and 0102, could you take a look at this patchset ?

**Comment by feiwan1** [comment]
LGTM

File: patches/0104-lavu-hwcontext_qsv-create-dynamic-frame-pool-if-requ.patch [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
++    if (ret < 0)
++        goto fail;
++
++#if CONFIG_VAAPI
++    if (child_frames_ctx->device_ctx->type == AV_HWDEVICE_TYPE_VAAPI) {
++        VASurfaceID *surface_id_internal;
++
++        surface_id_internal = av_calloc(1, sizeof(*surface_id_internal));
++        if (!surface_id_internal)
++            return NULL;
```
[the remark this answers is not in the document]
Comment (xhaihao) [reply patches/0104-lavu-hwcontext_qsv-create-dynamic-frame-pool-if-requ.patch]: Thanks for catching this, fixed in the new version

**Comment by wenbinc-Bin** [comment]
LGTM

**Pull request merged**